README.md

```diff
@@ -151,6 +151,24 @@ script:
 
 The monitoring dashboard is accessible at `http://kubernetes-master-1.kubernetes-playground.local/monitoring/cluster`
 
+### Kites experiments
+
+#### How to deploy the net-test-dev DaemonSet
+
+On the Master node, run:
+`kubectl apply -f /vagrant/kubernetes/kites/net-test-dev_ds.yaml`
+
+Check that the pods created are in the "Running" state with:
+`kubectl get pod -o wide`
+
+Finally, to enter the newly created pod you have to run:
```

Comment:
What to you mean by `enter`? Maybe `open a shell in the XXXX container` is better